custom_components/tuya_local/devices/pv28-cw_airquality_monitor.yaml

```diff
@@ -0,0 +1,299 @@
+    name: Alarm Switch
+    category: config
+    dps:
+      - id: 13
+        type: boolean
+        name: switch
+        mapping:
+          - dps_val: true
+            icon: "mdi:bell"
+          - dps_val: false
+            icon: "mdi:bell-off"
+  - entity: sensor
+    name: Power State
+    class: enum
+    category: diagnostic
+    dps:
+      - id: 14
+        type: string
+        name: sensor
+        mapping:
+          - dps_val: normal
+            value: battery
+          - dps_val: charge
+            value: charging
+  - entity: sensor
+    name: Battery
+    class: battery
+    category: diagnostic
+    dps:
+      - id: 15
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  # - entity: number
+  #   name: Screen Brightness
+  #   category: config
+  #   dps:
+  #     - id: 17
+  #       type: integer
+  #       name: value
+  #       unit: "%"
+  #       range:
+  #         min: 0
+  #         max: 100
+  #       mapping:
+  #         - step: 1
+  - entity: sensor
+    name: Temperature
+    class: temperature
+    dps:
+      - id: 18
+        type: integer
+        name: sensor
+        unit: "°C"
+        class: measurement
+  - entity: sensor
+    name: Humidity
+    class: humidity
+    dps:
+      - id: 19
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    name: PM 2.5
+    class: pm25
+    dps:
+      - id: 20
+        type: integer
+        name: sensor
+        unit: µg/m³
+        class: measurement
+  - entity: sensor
+    name: TVOC
+    class: volatile_organic_compounds
+    dps:
+      - id: 21
+        type: integer
+        name: sensor
+        unit: µg/m³
+        class: measurement
+        mapping:
+          - scale: 1
+            step: 1
+  - entity: sensor
+    name: HCHO
+    class: volatile_organic_compounds
+    dps:
+      - id: 22
+        type: integer
+        name: sensor
+        unit: µg/m³
+        class: measurement
+        mapping:
+          - scale: 1
+            step: 1
+  - entity: number
+    name: CO2 alarm threshold
+    category: config
+    dps:
+      - id: 26
+        type: integer
+        name: value
+        unit: "ppm"
+        range:
+          min: 800
+          max: 2000
+        mapping:
+          - step: 1
+  - entity: select
+    name: Temperature unit
+    category: config
+    dps:
+      - id: 31
+        type: string
+        name: option
+        mapping:
+          - dps_val: c
+            value: "°C"
+          - dps_val: f
+            value: "°F"
+  - entity: sensor
+    name: PM 10
+    class: pm10
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        unit: µg/m³
+        class: measurement
+  - entity: sensor
+    name: PM 1.0
+    class: pm1
+    dps:
+      - id: 102
+        type: integer
+        name: sensor
+        unit: µg/m³
+        class: measurement
+  - entity: sensor
+    name: Humidity
```

Comment:
There is already a sensor called humidity on dp 19.  This one needs a unique name to differentiate it.

Reply:
understood